### ISSUE TITLE
MRView: screen capture: Restore stack maintains capture parameters

### ISSUE THREAD

**Issue by rtabbara** [MRtrix3/mrtrix3#243]
Currently, the screen capture restore does not reset the capture parameter values.

**Comment by jdtournier** [comment]
It's just occurred to me that providing a full stack for undo/rewind opens a can of worms. For this to be done right, you'd probably also need to provide a 'redo' button (in case I accidentally hit rewind too many times). You'd need to properly reset the view and UI parameters in this case too. As things stand, the rewind stack opens the possibility of accidentally messing up a complex movie render if I hit rewind twice by mistake - I can't go forward in the stack. I think it's probably best we remove the rewind stack and just leave it as a single undo...

Thoughts...?

**Comment by rtabbara** [comment]
@jdtournier Fair enough. I'll just change the stack size to 1 in case in the future we want the ability to undo/redo multiple capture parameter changes.

**Comment by rtabbara** [comment]
Closing for now. Stack size set to 1 in f6c7149f96346f69b360f6f98e933b3f03087c63

**Comment by jdtournier** [comment]
Reopening - we may want to do something about this in future... 

My point above was really about the _current_ behaviour, which is now fixed.
